fcm_django/migrations/0011_fcmdevice_fcm_django_registration_id_user_id_idx.py

```diff
@@ -33,12 +35,16 @@ class Migration(migrations.Migration):
         ("fcm_django", "0010_unique_registration_id"),
     ]
 
-    operations = [
-        AddIndexSkipMySQL(
-            model_name="fcmdevice",
-            index=models.Index(
-                fields=["registration_id", "user"],
-                name="fcm_django__registr_dacdb2_idx",
+    operations = (
+        [
+            AddIndexSkipMySQL(
+                model_name="fcmdevice",
+                index=models.Index(
+                    fields=["registration_id", "user"],
+                    name="fcm_django__registr_dacdb2_idx",
+                ),
             ),
-        ),
-    ]
+        ]
+        if not SETTINGS["MYSQL_COMPATIBILITY"]
```

Comment:
Why explicitly check this? It should skip it if it's mysql anway?

Reply:
To make `makemigrations` happy:

1st execution of `makemigrations` is with the check.
2nd execution is without it.

![image](https://github.com/xtrinch/fcm-django/assets/2981274/3a3630a0-d2a7-4db8-acc4-c7caab8c8165)